docs/source/plotting.rst

```diff
@@ -251,10 +251,29 @@ method from there, directing it to the correct position with the ``loc`` and
     axes.iloc[ 0, -1].legend(loc='lower right',  bbox_to_anchor=(1, 1))
     axes.iloc[-1,  0].legend(loc='lower center', bbox_to_anchor=(len(axes)/2, len(axes)))
 
+Log-scale
+---------
+
+You can plot selected parameters on a log-scale by passing a list of those
+parameters under the keyword ``logx`` to :func:`anesthetic.plot.make_1d_axes`
+or :meth:`anesthetic.samples.Samples.plot_1d`, and under the keyword ``logxy``
+to :func:`anesthetic.plot.make_2d_axes` or
+:meth:`anesthetic.samples.Samples.plot_2d`:
+
+.. plot:: :context: close-figs
+
+    fig, axes = make_1d_axes(['x0', 'x1', 'x2'], logx=['x2'])
+    samples.plot_1d(axes, label="'x2' on log-scale")
+
+.. plot:: :context: close-figs
+
+    fig, axes = make_2d_axes(['x0', 'x1', 'x2'], logxy=['x2'])
+    samples.plot_2d(axes, label="'x2' on log-scale")
+
```

Comment:
New docs can be found here:
https://anesthetic.readthedocs.io/en/logscale/plotting.html#log-scale

Reply:
Nice